core/server/data/migrations/versions/3.0/1-add-type-column.js

```diff
@@ -0,0 +1,42 @@
+const common = require('../../../../lib/common');
+const commands = require('../../../schema').commands;
+
+const createLog = type => msg => common.logging[type](msg);
+
+function createColumnMigration({table, column, dbIsInCorrectState, operation}) {
+    return function columnMigrations({transacting}) {
+        return transacting.schema.hasColumn(table, column)
+            .then(dbIsInCorrectState)
+            .then((isInCorrectState) => {
+                const log = createLog(isInCorrectState ? 'warn' : 'info');
+
+                log(`Adding ${column} column to ${table} table`);
```

Comment:
The `Adding` bit won't be always true here as `down` migration is `Dropping`

Reply:
Also, think it's a little misleading message. When `isCorrectState` is `false` we "skip adding" the column altogether, so would rather give that on the output :thinking:

Reply:
Hmm, the misleading message was copied straight from the previous migration - I was under the impression that `warn` output in migrations implied that the operation didn't happen